noetic/distribution.yaml

```diff
@@ -2524,6 +2524,21 @@ repositories:
       type: git
       url: https://github.com/fetchrobotics/power_msgs.git
       version: ros1
+  py_trees_msgs:
+    doc:
+      type: git
+      url: https://github.com/splintered-reality/py_trees_msgs.git
```

Comment:
@stonier It looks like previous releases of this package were done from the `stonier` github organization.  This one looks to be released from `splintered-reality`.  That's totally fine, but should we update the older versions (in Melodic) to point to the splintered-reality organization instead?

Reply:
Thought about that, but should be fine with github redirects right?